cli/pcluster/config/validators.py

```diff
@@ -1304,11 +1314,13 @@ def fsx_ignored_parameters_validator(section_key, section_label, pcluster_config
 
     # If fsx_fs_id is specified, all parameters besides shared_dir are ignored.
     relevant_when_using_existing_fsx = ["fsx_fs_id", "shared_dir"]
+    PARAM_WITH_DEFAULT = {"drive_cache_type": "NONE"}
     if fsx_section.get_param_value("fsx_fs_id") is not None:
         for fsx_param in fsx_section.params:
-            if fsx_param not in relevant_when_using_existing_fsx and fsx_section.get_param_value(fsx_param) is not None:
+            if fsx_param not in relevant_when_using_existing_fsx and PARAM_WITH_DEFAULT.get(
```

Comment:
We don't need to define the `PARAM_WITH_DEFAULT`. We can replace: 
`PARAM_WITH_DEFAULT.get(fsx_param, None) != fsx_section.get_param_value(fsx_param)`
with:
`fsx_section.get_param(fsx_param).get_default_value() != fsx_section.get_param_value(fsx_param)`